server/game/effects.js

```diff
@@ -136,14 +136,15 @@ const Effects = {
             }
         };
     },
-    setSetupGold: function(value) {
+    setMaxBullets: function(value) {
```

Comment:
Should this function have a unit test associated with it?

Reply:
Yes, max bullets overall would probably deserve own unit test. Honestly, lot of effects are missing tests.

Reply:
Created a new issue to handle tests for effects: #193